content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -185,6 +185,11 @@ These instructions are for Kubernetes {{< skew currentVersion >}}.
    curl -fsSL https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/deb/Release.key | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
    ```
 
+   {{< note >}}
+   In releases older than Debian 12 and Ubuntu 22.04, `/etc/apt/keyrings` does not exist by default;
+   you can create it by running `sudo mkdir -m 755 /etc/apt/keyrings`
+   {{< /note >}}
+
```

Comment:
The page layout is broken due to this change _(indentation of the note block might be causing this)_. 
See the preview ([https://deploy-preview/tools/kubeadm/install-kubeadm/](https://deploy-preview-44671--kubernetes-io-main-staging.netlify.app/docs/setup/production-environment/tools/kubeadm/install-kubeadm/)) for more insights.

Reply:
Fair. A bit frustrating, but fair.